.editorconfig

```diff
@@ -57,3 +57,6 @@ csharp_new_line_before_members_in_anonymous_types = true
 
 # CS0168: Variable is declared but never used
 dotnet_diagnostic.CS0168.severity = suggestion
+
+# S3925: "ISerializable" should be implemented correctly
+dotnet_diagnostic.S3925.severity = none
```

Comment:
I don't see any `ISerializable` in this PR, so I don't think anything in here is affected by this change, but if you do want to turn off this rule completely we should also alter the [configuration for SonarCloud](https://github.com/SteeltoeOSS/steeltoe/blob/master/Steeltoe.ruleset#L269)

Reply:
Yes .. lets turn this off